apps/studio/src/server/modules/page/page.router.ts

```diff
@@ -405,7 +374,7 @@ export const pageRouter = router({
       if (isEmpty(permalinkTree)) {
         throw new TRPCError({
           code: "NOT_FOUND",
-          message: "No permalink could be found for the given page",
+          message: "No permalink tree could be found for the given page",
```

Comment:
sadly we should revert this because this error is likely going to be seen by the user and the place where it shows is in `/settings` for the permalink .-.